### PR TITLE
BIDS compliance of eyetracking data

### LINKED ISSUE #12: BIDS eyetracking data
The BEP20 extension proposal isn't yet merged (https://docs.google.com/document/d/1eggzTCzSHG3AEKhtnEDbcdk-2avXN6I94X8aUPEBVsw/edit#heading=h.9tphvz6ot0j1), but better than when I last looked at it. Its probably a good idea to check which metadata they recommend and require, and as far as I can see from a quick browse through BIDS 1.5.0, the json sidecar files for eyetracking data are yet missing.

### PR DESCRIPTION
**Pull request by adswa**
Fixes https://github.com/psychoinformatics-de/studyforrest-data/issues/12

### CODE REVIEW

File: .bidsignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+**/*.asc
```
Comment (loj) [inline .bidsignore]: Almost... it needs to be

`**/*.asc.gz`

**Review by loj — COMMENTED** [review]
Just a small change to the `.bidsignore` file. Otherwise things look good. Thanks!

File: .bidsignore (hunk 1 above)
Comment (loj) [reply .bidsignore]: Ah, and one other thing that could be added to the `.bidsignore` file:

`task-*_physio.json`

See https://github.com/bids-standard/bids-validator/issues/1275#issuecomment-827644778

Comment (adswa) [reply .bidsignore]: ahhhh crap, thanks for catching that

Comment (adswa) [reply .bidsignore]:
```suggestion
**/*.asc.gz
```

**Comment by adswa** [comment]
Should be good now, I applied both of your suggestions.

**Comment by loj** [comment]
Looks good. Thanks!

**Pull request merged**